misc/xml-to-record-converter/src/test/java/io/ballerina/xmltorecordconverter/XMLToRecordConverterTests.java

```diff
@@ -545,8 +558,35 @@ public void textXMLWithDefaultValueNode() throws IOException {
     public void textXMLWithDefaultValueNode2() throws IOException {
         String xmlFileContent = Files.readString(sample34XML);
         String generatedCodeBlock = XMLToRecordConverter.convert(xmlFileContent, false, false, false,
-                "__text", true).getCodeBlock().replaceAll("\\s+", "");
+                "__text", true, false, false).getCodeBlock().replaceAll("\\s+", "");
         String expectedCodeBlock = Files.readString(sample34Bal).replaceAll("\\s+", "");
         Assert.assertEquals(generatedCodeBlock, expectedCodeBlock);
     }
+
+    @Test(description = "textXMLWithoutAttributes")
+    public void textXMLWithoutAttributes() throws IOException {
+        String xmlFileContent = Files.readString(sample35XML);
+        String generatedCodeBlock = XMLToRecordConverter.convert(xmlFileContent, false, false, false,
+                null, true, true, false).getCodeBlock().replaceAll("\\s+", "");
+        String expectedCodeBlock = Files.readString(sample35Bal).replaceAll("\\s+", "");
+        Assert.assertEquals(generatedCodeBlock, expectedCodeBlock);
+    }
+
+    @Test(description = "textXMLWithoutAttributeAnnotation")
+    public void textXMLWithoutAttributeAnnotation() throws IOException {
+        String xmlFileContent = Files.readString(sample35XML);
+        String generatedCodeBlock = XMLToRecordConverter.convert(xmlFileContent, false, false, false,
+                "__text", false, false, true).getCodeBlock().replaceAll("\\s+", "");
+        String expectedCodeBlock = Files.readString(sample36Bal).replaceAll("\\s+", "");
+        Assert.assertEquals(generatedCodeBlock, expectedCodeBlock);
+    }
+
+    @Test(description = "textXMLWithoutMultipleAttributeAnnotation")
+    public void textXMLWithoutMultipleAttributeAnnotation() throws IOException {
+        String xmlFileContent = Files.readString(sample36XML);
+        String generatedCodeBlock = XMLToRecordConverter.convert(xmlFileContent, false, false, false,
+                "__text", false, false, true).getCodeBlock().replaceAll("\\s+", "");
+        String expectedCodeBlock = Files.readString(sample37Bal).replaceAll("\\s+", "");
+        Assert.assertEquals(generatedCodeBlock, expectedCodeBlock);
```

Comment:
Shall we rename the xml source and the generated bal with same name? here source is `sample_36.xml` and the generated bal file is `sample_37.bal`, which is confusing.

Reply:
Here, I used the same xml file for two test cases.